src/platform/AT32/timer_def.h

```diff
@@ -24,10 +24,10 @@
 #include "platform.h"
 #include "common/utils.h"
 
-#define USED_TIMERS  ( BIT(1) | BIT(2) | BIT(3) | BIT(4) | BIT(5) | BIT(8) | BIT(9) | BIT(10) | BIT(11) | BIT(12) | BIT(13) | BIT(14) | BIT(20) )
+#define USED_TIMERS  ( BIT(1) | BIT(2) | BIT(3) | BIT(4) | BIT(5) | BIT(6) | BIT(7) | BIT(8) | BIT(9) | BIT(10) | BIT(11) | BIT(12) | BIT(13) | BIT(14) | BIT(20) )
 #define TIMUP_TIMERS ( BIT(1) | BIT(2) | BIT(3) | BIT(4) | BIT(5) | BIT(8) | BIT(20) )
 #define FULL_TIMER_CHANNEL_COUNT        109
```

Comment:
This will trigger on bit errors, but it should be moved to some common place (even C timer init is fine)
```C
#if defined(USED_TIMERS)
    STATIC_ASSERT((~USED_TIMERS  & TIMUP_TIMERS) == 0, "All TIMUP timers must be used");
#endif
```